http_check/datadog_checks/http_check/http_check.py

```diff
@@ -85,21 +84,22 @@ def _check(self, instance):
         def send_status_up(logMsg):
             # TODO: A6 log needs bytes and cannot handle unicode
             self.log.debug(logMsg)
-            service_checks.append((self.SC_STATUS, Status.UP, "UP"))
+            service_checks.append((self.SC_STATUS, AgentCheck.UP, "UP"))
```

Comment:
1) AFAIU we always pass `SC_STATUS`, so you can remove it.
2) I don't think `AgentCheck.UP` is a thing.

Reply:
In line 240 we pass `self.SC_SSL_CERT`